web-frontend/src/pages/Settings/SettingsSignRule.tsx

```diff
@@ -1,16 +1,25 @@
+import { useEffect, useMemo, useState } from 'react';
+
 import {
   Button,
   ButtonIcon,
   ButtonToggle,
   Clipboard,
   FetchingLine,
   toast,
+  Tooltip,
 } from '@massalabs/react-ui-kit';
-import { FiTrash2 } from 'react-icons/fi';
+import { maskAddress } from '@massalabs/react-ui-kit/src/lib/massa-react/utils';
+import { MassaStationWallet } from '@massalabs/wallet-provider';
+import {
+  Config,
+  RuleType,
+  SignRule,
+} from '@massalabs/wallet-provider/dist/esm/massaStation/types';
```

Comment:
Do not import things from "dist".
If export are missing, you have to modify wallet-provider to export things properly